cmd/address.go

```diff
@@ -166,13 +166,13 @@ func (a *NetworkScan) InitAddressCommand() {
 	bruteForceCmd.Flags().StringSlice("passwords", []string{}, "Password to use in attack")
 	bruteForceCmd.Flags().StringSlice("usernamelists", []string{}, "File paths containing usernames to use in attack")
 	bruteForceCmd.Flags().StringSlice("passwordlists", []string{}, "File paths containing passwords to use in attack")
-	bruteForceCmd.Flags().Int("timeout", 3, "Timeout per request (Seconds)")
+	bruteForceCmd.Flags().Int("timeout", 3000, "Timeout per request (MilliSeconds)")
 	bruteForceCmd.Flags().Int("sleep", 3, "Sleep time between requests (Seconds)")
```

Comment:
lets not mix seconds and milliseconds, can you make sleep milliseconds too?

Reply:
once you do that I'll test once more, but things look good otherwise

its the sleep i wanted to make lower, because I want to send the requests faster

Reply:
If you are talking about Telnet this is because you need to have sleeps for this to work. Since there is a delay in when you enter a cmd and when a new prompt is received between each interaction there is a 1 second sleeps that not configured by the sleep flag. The sleep flag at the moment isnt used since its used for when we get ratelimited which isnt implement right now due to a lack of test envs

Reply:
ACK on change sleep to milliseconds

Reply:
When I had multithread implemented I ran the command and it ripped but on close investigation the commands werent being excuted in the desired way so I realized there is a cap to how fast we run run Telent to potentially have a login (For DoS though this goes out the window)

Reply:
Sorry not MultiThread

Reply:
Only way to get it to go fast is TO Multithread it, I forgot the Multithread DoSed my Docker thats why it didnt work. But the point still stands that we need the sleeps so single threading has a limit to the speed it can go